src/sql/workbench/contrib/objectExplorer/browser/serverTreeView.ts

```diff
@@ -619,6 +622,13 @@ export class ServerTreeView extends Disposable implements IServerTreeView {
 					if (errorListener) {
 						errorListener.dispose();
 					}
+
+					this._telemetryService.createActionEvent(TelemetryKeys.TelemetryView.ObjectExplorer, TelemetryKeys.TelemetryAction.ObjectExplorerFilter)
+						.withAdditionalProperties({
+							filterPropertyNames: JSON.stringify(filters.map(f => f.displayName)),
```

Comment:
You shouldn't use the display names since those are localized. There's a non-localized name field on these, right?

Reply:
Unfortunately, I haven't added it to the release yet.

Reply:
Hm, ok well please make an issue to track adding that and updating this for follow up next release then.

Reply:
Actually, I had added it to sts but never used in ads. I just checked and the filter properties do have a non localized 'name' property. I have added that to the telemetry event. For the next release, I will switch to using name instead of localized names.